lib/view_model/send/send_item.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:cake_wallet/bitcoin/bitcoin_amount_format.dart';
+import 'package:cake_wallet/bitcoin/electrum_wallet.dart';
+import 'package:cake_wallet/entities/calculate_fiat_amount_raw.dart';
+import 'package:cake_wallet/entities/parse_address_from_domain.dart';
+import 'package:cake_wallet/entities/parsed_address.dart';
+import 'package:cake_wallet/monero/monero_amount_format.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:mobx/mobx.dart';
+import 'package:cake_wallet/core/wallet_base.dart';
+import 'package:cake_wallet/monero/monero_wallet.dart';
+import 'package:cake_wallet/entities/calculate_fiat_amount.dart';
+import 'package:cake_wallet/entities/wallet_type.dart';
+import 'package:cake_wallet/store/dashboard/fiat_conversion_store.dart';
+import 'package:cake_wallet/store/settings_store.dart';
+import 'package:cake_wallet/generated/i18n.dart';
+
+part 'send_item.g.dart';
+
+const String cryptoNumberPattern = '0.0';
+
+class SendItem = SendItemBase with _$SendItem;
+
+abstract class SendItemBase with Store {
```

Comment:
"SendItem" is not best name, let use just "Output" for now ("outputs" for list of Output objects)